File: app/src/main/java/com/zzzmode/appopsx/ui/main/backup/RestoreModel.java
package com.zzzmode.appopsx.ui.main.backup;

import com.zzzmode.appopsx.ui.model.PreAppInfo;
import java.util.List;

/**
 * Created by zl on 2017/5/7.
 */

class RestoreModel {

  long createTime;
  int version;
  int size;
  long fileSize;
  String path;
  String fileName;
  List<PreAppInfo> preAppInfos;
}

File: app/src/main/kotlin/com/zzzmode/appopsx/ui/main/backup/RestoreModel.kt
package com.zzzmode.appopsx.ui.main.backup

import com.zzzmode.appopsx.ui.model.PreAppInfo

/**
 * Created by zl on 2017/5/7.
 */

internal class RestoreModel {

    var createTime: Long = 0
    var version: Int = 0
    var size: Int = 0
    var fileSize: Long = 0
    var path: String? = null
    var fileName: String? = null
    var preAppInfos: List<PreAppInfo>? = null
}
